pages/common/tldr.md

```diff
@@ -5,3 +5,7 @@
 - Get typical usages of a command (hint: this is how you got here!):
 
 `tldr {{command}}`
+
+- For typical usages of a subcommand:
+
+`tldr {{command-subcommand}}`
```

Comment:
This is not true. You don't need the `-`. You can do "tldr git merge" and it works fine.

Reply:
```bash
$ tldr git merge
Invalid argument `merge'

Usage: tldr [-v|--version] [--update] COMMAND
  tldr Client program

```

Ubuntu 18.10

Reply:
That is a client specific thing. The node client handles it fine. Since this is not in any spec, I don't think we should mention it.

Reply:
So does the bash client. Perhaps you could open an issue against your client's repo?

_(For my own reference, this is worth including in the official client spec that's being drafted in #1065)_